pkg/pushsync/pushsync.go

```diff
@@ -126,19 +130,34 @@ func (ps *PushSync) handler(ctx context.Context, p p2p.Peer, stream p2p.Stream)
 		return swarm.ErrInvalidChunk
 	}
 
-	span, _, ctx := ps.tracer.StartSpanFromContext(ctx, "pushsync-handler", ps.logger, opentracing.Tag{Key: "address", Value: chunk.Address().String()})
-	defer span.Finish()
-
-	// Get price we charge for upstream peer read at headler
```

Comment:
this wasnt a typo really, we read price from our own pricetable in headler, and use it in handler